content.org

```diff
@@ -31,8 +24,21 @@
 
 - Also, thanks to nilenso, IN/Clojure and ClojureBridge core team.
 
-- https://tinyurl.com/clj-bridge-blr
--> outline.org
+- https://nilenso.com/clojurebridge
+
+** Speakers
+
+*** Soumya Maithani
+Currently working as a senior platform developer at Helpshift, developing
+in Clojure for the past 9 years and hoping to continue for more.
+I love distributed systems, event streaming and exploring new databases.
+In my spare time, I love reading fiction, cooking and am learning football.
+
+*** Sezal Jain
+
+# TODO 
```

Comment:
Intro for Sezal is pending

Reply:
Yeah, I haven't received a bio from her yet.

Reply:
Have added a bio.